junit5/src/test/java/cz/xtf/junit5/listeners/ManagedBuildPrebuilderTest.java

```diff
@@ -0,0 +1,116 @@
+package cz.xtf.junit5.listeners;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import cz.xtf.core.bm.BinaryBuild;
+import cz.xtf.core.config.XTFConfig;
+import cz.xtf.junit5.annotations.SinceVersion;
+import cz.xtf.junit5.annotations.SkipFor;
+import cz.xtf.junit5.annotations.UsesBuild;
+import cz.xtf.junit5.interfaces.BuildDefinition;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
+import uk.org.webcompere.systemstubs.properties.SystemProperties;
+
+@ExtendWith(SystemStubsExtension.class)
+class ManagedBuildPrebuilderTest {
+
+    @SystemStub
+    private SystemProperties systemProperties;
+
+    @BeforeEach
+    public void before() {
+        systemProperties.set("xtf.eap.subid", "test");
+        systemProperties.set("xtf.eap.test.image", "quay.io/wildfly/wildfly-s2i-jdk11:26.0");
```

Comment:
Optional: Maybe adding a comment here that if wildfly version is updated then tests need to be updated as well.

Reply:
comment added